app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageViewModel.kt

```diff
@@ -98,6 +98,9 @@ class NewMessageViewModel @Inject constructor(
     private val ioCoroutineContext = viewModelScope.coroutineContext(ioDispatcher)
 
     var draftInRAM: Draft = Draft()
+
+    val aiOutputToInsert = SingleLiveEvent<Pair<String?, String>>()
```

Comment:
Why did you need to move `aiOutputToInsert` from AiViewModel to NewMessageViewModel? Can it not stay there?